### PR TITLE
Make (map)-ideas and topicprio moderateble (Ampel)

### PR DESCRIPTION
**Pull request by kleingeist**
This PR adds the moderatable variant (ampel) to the following apps (as defined in the US)
* ideas  (Brainstorming, Idea Collection, AgendaSettings)
* mapideas  (Spatial Brainstorming, Spatial Idea Collection)
* topicprio  (Topic Prioritization)

Note that **maptopicprio (Place Prioritization) is not moderateble yet**. As bürgerhaushalt and kiezkasse  were moderateble before it is the only app/process left
@CarolingerSeilchenspringer should the maptopicprio have the option to give moderator feedback, to?

### CODE REVIEW

**Comment by kleingeist** [comment]
I could move the Moderateble to the `AbstractIdea` model as every model inheriting from it is using the moderateable, too. But  i am not sure if this is a good idea, as this limits our possibilities if we have to introduce a new itemlike app

what do you think?

**Comment by fuzzylogic2000** [comment]
I think it is a good idea to move it to the AbstractIdea model. I think in most cases the new app would also need it anyway. And if not, we can either split it up again or make a new model.

And I think, we can also make the maptopicprio moderatable right away? It doesn't make a lot of sense to use it everywhere but there.

File: meinberlin/apps/contrib/templatetags/item_tags.py [hunk 1]
```diff
@@ -44,14 +45,19 @@ def get_item_delete_url(item):
...
-    return reverse(url_name, kwargs={
-        'year': item.created.year,
-        'pk': '{:05d}'.format(item.pk)
-    })
+    try:
+        return reverse(url_name, kwargs={
+            'year': item.created.year,
+            'pk': '{:05d}'.format(item.pk)
+        })
+    except NoReverseMatch:
```
Comment (fuzzylogic2000) [inline meinberlin/apps/contrib/templatetags/item_tags.py]: Nice!

**Comment by kleingeist** [comment]
i think the migration is missing in the commit

**Comment by CarolingerSeilchenspringer** [comment]
we don't need the Ampel for Topic- & Placeprio! BUT: For Story #165-[mB] internal categories for moderation we need moderation option for ALL ideas and comments.

**Comment by kleingeist** [comment]
ups. i will remove the ampel from (map)topicprio again. i have no idea how i came up with the idea

**Pull request merged**